src/Network/NWPathMonitor.cs

```diff
@@ -22,9 +22,17 @@ public class NWPathMonitor : NativeObject {
 		[DllImport (Constants.NetworkLibrary)]
 		extern static IntPtr nw_path_monitor_create ();
 
+		public NWPath currentPath;
```

Comment:
Use a get only property with a private var.
```suggestion
		private NWPath currentPath;
		public NWPath CurrentPath => currentPath;
```

Reply:
i think it is private by default, so they private keyword isn't required

Reply:
fixed